components/sup/src/manager/service.rs

```diff
@@ -53,8 +53,11 @@ pub use habitat_common::templating::{config::{Cfg,
                                                PkgProxy}};
 use habitat_common::{outputln,
                      templating::{config::CfgRenderer,
-                                  hooks::Hook,
-                                  package::DEFAULT_USER}};
+                                  hooks::Hook}};
+
+#[cfg(windows)]
+use habitat_common::templating::package::DEFAULT_USER;
```

Comment:
This and some of the unrelated looking changes are to get rid of warnings when compiling on linux.